venues/NIPS.cc/2017/Workshop/Autodiff/params.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do you need this file and the file /data/NIPS2017_autodiff.json ? they do look different but they look like they're achieving similar things...

Reply:
They are the files used by admin/superuser_init.py to create the conferences.  So yes they are very similar, but different titles and due dates.

Reply:
I still don't fully understand why we need both of them. The `conference_title`s look the same to me... am I missing something?